skbio/io/registry.py

```diff
@@ -558,11 +558,25 @@ def _init_reader(self, file, fmt, into, verify, kwargs, io_kwargs):
 
         reader = self.get_reader(fmt, into)
         if reader is None:
+            readers = self._get_possible_readers(fmt)
+            possible_intos = [r.__name__ for r in readers] if readers else None
+            if possible_intos:
```

Comment:
Oh I see, let's make it so that `_get_possible_readers` always returns a list (empty if there isn't anything). That way we can just chain it with the list comprehension. Then `possible_intos` will be empty (false) or it will have the available class names.